multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -147,6 +147,15 @@ def siamese_network():
     return automm_presets.create("default")
 
 
+@automm_presets.register()
+def text_similarity():
+    return {
+        "model.names": ["hf_text"],
+        "model.hf_text.checkpoint_name": "sentence-transformers/all-mpnet-base-v2",
```

Comment:
consider use "all-MiniLM-L6-v2" which is way faster

Reply:
Sure. Changed.